drivers/clock-distribution/ltc6953/ltc6953.c

```diff
@@ -0,0 +1,780 @@
+				       LTC6953_ADEL(delay));
+	case 4:
+		return ltc6953_update (dev, LTC6953_REG(0x1F), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 5:
+		return ltc6953_update (dev, LTC6953_REG(0x23), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 6:
+		return ltc6953_update (dev, LTC6953_REG(0x27), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 7:
+		return ltc6953_update (dev, LTC6953_REG(0x2B), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 8:
+		return ltc6953_update (dev, LTC6953_REG(0x2F), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 9:
+		return ltc6953_update (dev, LTC6953_REG(0x33), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	case 10:
+		return ltc6953_update (dev, LTC6953_REG(0x37), LTC6953_ADEL_MSK,
+				       LTC6953_ADEL(delay));
+	default:
+		return -EINVAL;
+	}
+
+}
+
+/**
+ * @brief Set LTC6953 output channel mode.
+ * @param dev - The device structure.
+ * @param channel - Output channel [0-10]
+ * @param mode - Output channel mode [0-3]
+ * @return Returns 0 in case of success or negative error code.
+ */
+int ltc6953_set_mode (struct ltc6953_dev *dev, int32_t channel, uint8_t mode)
+{
+
+	if ((mode > 3) || (mode < 0)) {
+		return -EINVAL;
+	}
+
+	switch (channel) {
+	case 0:
+		return ltc6953_update (dev, LTC6953_REG(0x0D), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 1:
+		return ltc6953_update (dev, LTC6953_REG(0x11), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 2:
+		return ltc6953_update (dev, LTC6953_REG(0x15), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 3:
+		return ltc6953_update (dev, LTC6953_REG(0x19), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 4:
+		return ltc6953_update (dev, LTC6953_REG(0x1D), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 5:
+		return ltc6953_update (dev, LTC6953_REG(0x21), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 6:
+		return ltc6953_update (dev, LTC6953_REG(0x25), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 7:
+		return ltc6953_update (dev, LTC6953_REG(0x29), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 8:
+		return ltc6953_update (dev, LTC6953_REG(0x2D), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 9:
+		return ltc6953_update (dev, LTC6953_REG(0x31), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	case 10:
+		return ltc6953_update (dev, LTC6953_REG(0x35), LTC6953_MODE_MSK,
+				       LTC6953_MODE(mode));
+	default:
+		return -EINVAL;
+	}
+}
+
+/**
+ * @brief Set LTC6953 output channel inversion.
+ * @param dev - The device structure.
+ * @param channel - Output channel [0-10].
+ * @param is_invert - Inversion setting.
+ * @return Returns 0 in case of success or negative error code.
+ */
+int ltc6953_enable_output_invert (struct ltc6953_dev *dev, int32_t channel,
+				  bool is_invert)
+{
+	switch (channel) {
+	case 0:
+		return ltc6953_update (dev, LTC6953_REG(0x0D), LTC6953_OINV_MSK,
```

Comment:
remove space between `ltc6953_update` and `(dev, LTC6953_REG(0x0D), LTC6953_OINV_MSK,`. This applies throughout the entire code.